engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -2405,6 +2540,46 @@ private LogicalOperatorInfo fillLogicalOperatorInfo(OperatorMeta operator)
     return loi;
   }
 
+  private LogicalModuleInfo fillLogicalModuleInfo(ModuleMeta module, boolean flatten, String parentModuleName)
+  {
+    LogicalModuleInfo lmi = new LogicalModuleInfo();
+    lmi.name = module.getName();
+    lmi.className = module.getModule().getClass().getName();
+    if (flatten) {
+      for (OperatorMeta operatorMeta : getLogicalPlan().getAllOperators()) {
```

Comment:
Instead of iterating over all the operators, can we not use `module.getDag().getAllOperators()`? Basically iterate over operators belonging to that module only?